### DISCUSSION TITLE
How are updates handled

### DISCUSSION THREAD

**Discussion by KrysRed** [NUWCDIVNPT/stig-manager#911] [Q&A]
My team has started exploring STIG Manager and would like to know how the tool updates are handled.  Will we be notified through the client, or through GitHub? When an update does come out, will it merge with our current deployment, or will data need to be restored after the update?

**Comment by cd-rite** [comment]
Hi @KrysRed You can "Watch" the repo (upper right of the GitHub page) and set it to notify you of new Releases with their "Custom" option. 
For updates, simply replace the old Release with the new Release. In cases where there has been a Database change, the new Release will make the required changes to the database when it first starts up.  (As always, ensuring you have good, tested database backups is highly recommended)

![image](https://user-images.githubusercontent.com/61710958/214881078-e1a259a6-4d01-4c7c-b4a9-bf78363a21f3.png)
